docs/platform/howto/vpc-peering-gcp.rst

```diff
@@ -7,25 +7,29 @@ Once you've created a :doc:`VPC on the Aiven platform <manage-vpc-peering>`, you
 
 2. Select **VPC Networks** and find the VPC that you want to connect to.
 
-3. Click the project name and make note of the *Project ID*.
+3. Click the project name, and make note of the **Project ID**.
 
-4. Under *VPC Networks*, make note of the VPC Network Name.
+4. Under **VPC Network**, make note of the **VPC Network Name**.
 
 5. In `Aiven Console <https://console.aiven.io>`_, select **VPCs** from the sidebar on the **Services** page.
 
 6. On the **Virtual private clouds** page, select the VPC connection that you created.
 
-7. On the **VPC Peering connections** page, enter the project ID and VPC network name from GCP, then click **Add peering connection**.
+7. In the **VPC Peering connections** view, enter the GCP project ID (step 3) into the **GCP project ID** field and the exact GCP VPC network name (step 4) into the **GCP VPC network name** field. Next, select **Add peering connection**.
 
    This adds a new connection with the *Pending Peer* status.
 
    .. note::
 
-      Click the blue Pending peer icon and make a note of the Aiven project name and the network name.
+      Select the blue **Pending peer** icon, and make a note of the Aiven project ID and the VPC network name.
 
-8. In your GCP Console, go to **VPC** > **VPC network peering** and select **Create Connection**.
+8. In your GCP Console, go to **VPC** > **VPC network peering**, and select **Create peering connection**.
 
-9. Enter a name for the peering connection, and enter the same project ID and network name that you found in step 7.
+9. To create a peering connection, take the following steps:
+
+    1. Enter a name for the peering connection.
+    2. Under **Peered VPC network**, select **In another project**.
+    3. Enter the Aiven project ID and the VPC network name identified in step 7 in Aiven Console.
```

Comment:
The indent here is too deep. Please you the 3-chars indent coz otherwise it renders incorrectly on the docs website.